src/libsyntax/parse/diagnostics.rs

```diff
@@ -1079,21 +1079,21 @@ impl<'a> Parser<'a> {
         }
     }
 
-    // Eat tokens until we can be relatively sure we reached the end of the
-    // statement. This is something of a best-effort heuristic.
-    //
-    // We terminate when we find an unmatched `}` (without consuming it).
-    crate fn recover_stmt(&mut self) {
+    /// Eats tokens until we can be relatively sure we reached the end of the
+    /// statement. This is something of a best-effort heuristic.
+    ///
+    /// We terminate when we find an unmatched `}` (without consuming it).
+    pub fn recover_stmt(&mut self) {
```

Comment:
Why this visibility change?

Reply:
Heh. I need that for my REPL. Should have been in another PR, but let's just keep it in here for now, okay?

Reply:
I'd like to avoid semantic changes in this PR. Please move it to a different PR where text is also added to the doc comment that explains what code relies on this being `pub`.